common/cards/season-x/hermits/skizzleman-rare.ts

```diff
@@ -33,58 +34,54 @@ class SkizzlemanRare extends Card {
 		},
 	}
 
+	newGasLightAttack(component: CardComponent, target: RowEntity) {
+		return {
+			attacker: component.entity,
+			target: target,
+			type: 'secondary',
+			log: (values) => `${values.target} took ${values.damage} damage from $vGas Light$`,
+		} satisfies AttackDefs
+	}
+
 	override onAttach(game: GameModel, component: CardComponent, observer: ObserverComponent) {
-		const {player} = component
+		const {player, opponentPlayer} = component
 
 		let attackedEntities = new Set<RowEntity>()
 		let usedSecondary = false
 
+		observer.subscribe(player.hooks.onTurnStart, () => {
+			attackedEntities = new Set()
+			usedSecondary = false
+		})
+
 		observer.subscribe(player.hooks.onAttack, (attack) => {
 			if (attack.isAttacker(component.entity) && attack.type === 'secondary') {
 				usedSecondary = true
 			}
+		})
 
+		observer.subscribe(opponentPlayer.hooks.onDefence, (attack) => {
 			// Status effect attacks have a special case because they happen at the end of the turn
-			if (attack.type === 'status-effect') {
-				if (attack.targetEntity && attackedEntities.has(attack.targetEntity)) return
-				attack.addDamage(component.entity, 20)
+			if (attack.type === 'status-effect' && attack.targetEntity) {
+				if (attackedEntities.has(attack.targetEntity)) return
```

Comment:
Does not filter out active Hermit, may need clarification on interactions with Fire/Poison and Knockback/Egg
```suggestion
				if (attackedEntities.has(attack.targetEntity) || opponentPlayer.activeRowEntity === attack.targetEntity) return
```

Reply:
I see. I believe I misunderstood the card.